data/map planets.txt

```diff
@@ -3008,32 +3029,38 @@ planet "New Tibet"
 	security 0.1
 
 planet "New Tortuga"
-	attributes core "core pirate" factory pirate religious
+	# Although New Tortuga is not a pirate-run planet, there are pirates, so it has to retain the pirate-related attributes.
+	# The planet is self-sufficient, hence the many other attributes.
+	attributes core "core pirate" factory pirate religious fishing research mining "slavery ban" "pirate-friendly" "humanika rafeki"
```

Comment:
I think adding "north pirate" makes sense as they have easy access to the "back way" through uninhabited space, but adding "south pirate" to enable those missions does not.  You can't even cross any of the specified distances to the origin/destinations intended by those missions before FW.

```suggestion
	attributes core "core pirate" "north pirate" factory pirate religious fishing research mining "slavery ban" "pirate-friendly" "humanika rafeki"
```

Reply:
No, it doesn't make sense to give New Tortuga "north pirate" when Stormhold lacks that:

```ruby
planet Stormhold
        attributes core "core pirate" forest frontier pirate
```

Reply:
True.  I misunderstood your intentions in adding "Humanika Refi Port" everywhere, as you later clarified in the discussion.